analyses/focal-cn-file-preparation/run-prepare-cn.sh

```diff
@@ -19,92 +19,80 @@ script_directory="$(perl -e 'use File::Basename;
   print dirname(abs_path(@ARGV[0]));' -- "$0")"
 cd "$script_directory" || exit
 
+
+GENCODE="ftp.ebi.ac.uk/pub/databases/gencode/Gencode_human/release_38/gencode.v38.annotation.gtf.gz"
```

Comment:
if we want to add this - maybe set it to download to `data` directory if it does not exist and the read from the data directory. So that we do not need to modify this script again.

Reply:
@runjin326 I think it would make more sense to completely remove the download lines. I suggest we leave this the way it is for now, and open another PR once the v11 set is ready.